File: iniciante/javascript/1009.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let nomeVendedor = lines.shift();
let salarioFixo = Number(lines.shift());
let montanteVendas = Number(lines.shift());
const porcentagemVendas = 0.15

const calcularSalario = (a, b) => (a + b * porcentagemVendas)

console.log(`TOTAL = R$ ${calcularSalario(salarioFixo, montanteVendas).toFixed(2)}`)

File: iniciante/python/1009.py
# -*- coding: utf-8 -*-

nome_vendedor = input()
salario_fixo = float(input())
total_vendas_mes = float(input())

total_receber = salario_fixo + 0.15 * total_vendas_mes

print(f'TOTAL = R$ {total_receber:.2f}')
